.npmrc

```diff
@@ -0,0 +1 @@
+message = "Version %s"
```

Comment:
The .editorconfig is designed (among other things) to avoid this kind of EOF inconsistency. Did you add a plugin?

Reply:
The plugin helped a lot, thanks!